pages/upload/upload.view.scss

```diff
@@ -0,0 +1,158 @@
+.upload-page {
+	@extend %display-flex;
+	@extend %align-items-flex-start;
+	background-color: $background;
+	font-size: 1.3em;
+    height: 100%;
+	margin-top: $topbarHeight;
+	width: 100%;
+	@include desktop {
+		box-sizing: border-box;
+    	padding: 16px;
+    }
+
+	.container {
+		@extend %display-flex;
+	    @extend %flex-direction-column;
+    	margin: 0 auto;
+    	min-height: 25vw;
+    	max-width: 1200px;
+		width: 100%;
+
+		@include desktop {
```

Comment:
It's not possible to include @extend directives in an @media and... due to we're @include a @media, this is not allowed